packages/core-data/src/batch-processor.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * WordPress dependencies
+ */
+import apiFetch from '@wordpress/api-fetch';
+
+export default async function batchProcessor( requests ) {
```

Comment:
The only really comment I have is that this would be helpful to anyone who wants to use the REST API batching, not just those using the `core-data` package. So it'd be nice if it would live elsewhere, though I'm not sure where.

Reply:
Yeah, totally. Going to leave it as an experimental API in `core-data` for now though while the API matures. Maybe one day we'll do something like this https://github.com/WordPress/gutenberg/pull/28210#discussion_r559511878.